FIDO2_messages_details.md

```diff
@@ -0,0 +1,188 @@
+## FIDO2 Authenticate Credential Request
+
+This message is sent as a request by the AUX MCU to authenticate a FIDO2 credential. The MAIN MCU responds
+with [FIDO2 Authenticate Credential Response](FIDO2_messages_details.md#fido2-authenticate-credential-response). The message is used to
+check whether the authenticator has the specified credential or not.
```

Comment:
2 different lines? :D

Reply:
What do you mean?

Reply:
on github it is showed to me:
used to << line break >>
check...